ops/helmfiles/dagster/helmfile.yaml

```diff
@@ -27,20 +27,28 @@ releases:
           serviceAccount:
             enabled: true
             name: monster-dagster
+      - runLauncher:
+          config:
+            k8sRunLauncher:
+              envConfigMaps:
+                # the k8s run launcher doesn't allow us to directly set a list of env vars, so we just
+                # point it at the env config map the daemon generates as a workaround. this will be the
+                # same set of custom env vars as every other component of the system.
+                - name: monster-dagster-daemon-env
       - userDeployments:
           # "user code" we deploy to dagster; these are dagster repositories of pipelines
           enabled: true
           deployments:
-            # if you add a new deployment here, make sure it's also reflected in the env files (environment/*.yaml)
-            # so that env vars get propagated to it correctly
             - name: "monster-hca-ingest"
               image:
                 repository: "us.gcr.io/broad-dsp-gcr-public/monster-hca-dagster"
-                tag: {{ requiredEnv "GIT_SHORTHASH" }}
+                tag: {{ requiredEnv "GIT_SHORTHASH" | quote }}
```

Comment:
learned the hard way that if we don't do this, certain hex combos will be parsed as numbers in very silly ways (2e34677 is 2 to the 34677th power, for instance)